scene_select/ard_scene_select.py

```diff
@@ -429,6 +450,11 @@ def make_ard_pbs(level1_list, workdir, **ard_click_params):
 )
 @click.option("--pkgdir", type=click.Path(file_okay=False, writable=True), help="The base output packaged directory.")
 @click.option("--env", type=click.Path(exists=True, readable=True), help="Environment script to source.")
+@click.option(
+    "--index-datacube-env",
+    type=click.Path(exists=True, readable=True),
+    help="Datacube indexing environment. Add this to index the ARD results.",
+)
```

Comment:
Is this correct? The name doesn't sound like a path.... unless it's a raw module path? 

I assume this is a wagl option, which I'm unknowledgeable about, but the help text could be clearer on what it expects.

Reply:
Thanks. I'll beef up the description.